src/vs/editor/browser/controller/editContext/native/nativeEditContext.ts

```diff
@@ -167,6 +167,7 @@ export class NativeEditContext extends AbstractEditContext {
 		// Force blue the dom node so can write in pane with no native edit context after disposal
 		this.domNode.domNode.blur();
 		this.domNode.domNode.remove();
+		this.textArea.domNode.remove();
 		super.dispose();
```

Comment:
nit: I prefer this approach so we don't need to override:

```
this._register(toDisposable(() => {
	// Force blue the dom node so can write in pane with no native edit context after disposal
	this.domNode.domNode.blur();
	this.domNode.domNode.remove();
	this.textArea.domNode.remove();
}));
```

Reply:
Ah yep thanks for the advice, indeed it removes the override 🤔